packages/stark-ui/src/modules/input-mask-directives/directives/text-mask-config.intf.ts

```diff
@@ -13,6 +8,10 @@ export interface StarkTextMaskBaseConfig {
 	 */
 	guide?: boolean;
```

Comment:
The documentation about the `guide` property is outdated (it points to angular2-text-mask).
Could you update this documentation ?